README.md

```diff
@@ -24,8 +24,129 @@ Please open a github issue.
 
 ## Getting Started
 
-....
+### How to Integrate
```

Comment:
Do we have pip requirements or they all coming from the project already?

Reply:
`pytest` is going to be a requirement I'd imagine, want me to add a blurb?

Reply:
I think it automatically gets installed as a sub-dependency